webpack.common.config.ts

```diff
@@ -617,7 +619,29 @@ module.exports = {
     }),
     new ForkTsCheckerWebpackPlugin({
       checkSyntacticErrors: true
-    })
+    }),
+    new webpack.LoaderOptionsPlugin({
+      options: {
+        macros: {
+          load: function(resourcePath, args, root) {
+            if (root === undefined) {
+              root = path.resolve(__dirname, 'core/templates/dev/head');
+            }
+            var objExtend = function(args, obj) {
+              args = Array.prototype.slice.call(args);
+              var _a = args.slice(1);
+              _a.unshift(Object.assign(obj, args[0]));
+              return _a;
+            };
+            var argsExpr = args ? '(' + objExtend + ')' + '(arguments, ' +
+              JSON.stringify(args) + ')' : 'arguments';
+
+            return 'require(' + JSON.stringify(loaderUtils.urlToRequest(
+              resourcePath, root)) + ').apply(null,' + argsExpr + ')';
+          }
```

Comment:
Could you move this to a separate file? Here then create two methods (`loadExtensions`, `load`) with different roots, one for loading files from `extensions/` and the other from `core/templates/dev/head`.

Reply:
Done